CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 **User improvements:**
 
-- The built-in `sqlite3** module of Python is now enabled.
+- Support for built-in modules:
+  - `sqlite3`
+  - `bz2`
+  - `mmap`
```

Comment:
Probably better to put this in the PR that adds support for those (https://github.com/iodide-project/pyodide/pull/364)

Reply:
Thanks! Updated.